src/Tizen.Uix.Tts/Tizen.Uix.Tts/TtsClient.cs

```diff
@@ -230,6 +230,67 @@ public enum ServiceState
         Unavailable
     };
 
+    /// <summary>
+    /// Enumeration for the playing mode of TTS.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    public enum PlayingMode
+    {
+        /// <summary>
+        /// Mode for TTS playing on TTS service.
+        /// </summary>
+        ByService = 0,
+
+        /// <summary>
+        /// Mode for TTS playing on TTS client.
+        /// </summary>
+        ByClient = 1
+    };
+
+    /// <summary>
+    /// Enumeration for the audio type.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    public enum AudioType
+    {
+        /// <summary>
+        /// Signed 16-bit audio type.
+        /// </summary>
+        RawS16 = 0,
+
+        /// <summary>
+        /// Unsigned 8-bit audio type.
+        /// </summary>
+        RawU8 = 1
+    };
+
+    /// <summary>
+    /// Enumeration for the synthesized pcm event.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    public enum SynthesizedPcmEvent
+    {
+        /// <summary>
+        /// Event when the synthesized PCM is failed.
```

Comment:
I'm not sure if I understood the implemented behavior, but I would like the description to be changed. I assume that a series of event will be fired in response to a utterance requested. Maybe like this:
``` cs
/// The PCM synthesis failed.
/// Received initial synthesized PCM data (optionally add a word like: portion, block, fragment) for an utterance.
/// Received synthesized PCM data, not the first and not last in the stream.
/// Received the final synthesized PCM data.

Reply:
Thank you. I think you understand exactly, so I fixed it as you mentioned.